osu.Game.Rulesets.Osu/UI/OsuResumeOverlay.cs

```diff
@@ -178,8 +186,8 @@ public OsuResumeOverlayInputBlocker()
 
             public bool OnPressed(KeyBindingPressEvent<OsuAction> e)
             {
-                bool block = BlockNextPress;
-                BlockNextPress = false;
+                bool block = NextPressToBlock == e.Action;
+                NextPressToBlock = null;
```

Comment:
Does changing this from boolean to `OsuAction` affect behaviour? or is it just a code quality change.

Reply:
I saw it as a reliability change because this component was not apparently ensuring that it's blocking the right input. I don't particularly have an example of the old code being broken but I think the new is safer.